File: JavaScript/Week-02/Day-2/AppendLetter.ts
'use strict';
// Create a function called "appendA()" that adds "a" to every string in the far list.
// The parameter should be a list.


function appendA(array: string[], stringToAdd: string) {
  for (let x: number = 0; x < array.length; x++) {
    array[x] = array[x].concat(stringToAdd)
  }
  return array;
}

let far: string[] = ['bo', 'anacond', 'koal', 'pand', 'zebr'];

console.log(appendA(far, 'a'));

// The output should be: 'boa', 'anaconda', 'koala', 'panda', 'zebra'

File: Java/Week2/Day2/AppendLetter.java
import java.util.*;

public class AppendLetter {
  public static void main(String... args) {
    List<String> far = Arrays.asList("bo", "anacond", "koal", "pand", "zebr");
    // Create a method called "appendA()" that adds "a" to every string in the "far"
    // list.
    // The parameter should be a list.

    System.out.println(appendA(far));
  }

  public static String appendA(List<String> far) {

    for (String element : far) {
      element+="a";
      System.out.println(element);
    }
    return "";
  }
}

// The output should be: "boa", "anaconda", "koala", "panda", "zebra"